src/util/parsing/lines_and_interfaces.jl

```diff
@@ -121,11 +126,63 @@ function make_pras_interfaces(
     interface_forward_capacity_array = Matrix{Int64}(undef, num_interfaces, s2p_meta.N)
     interface_backward_capacity_array = Matrix{Int64}(undef, num_interfaces, s2p_meta.N)
 
-    for i in 1:num_interfaces
-        interface_forward_capacity_array[i, :] =
-            sum(line_forward_cap[interface_line_idxs[i], :], dims=1)
-        interface_backward_capacity_array[i, :] =
-            sum(line_backward_cap[interface_line_idxs[i], :], dims=1)
+    area_interchanges = collect(PSY.get_components(PSY.AreaInterchange, sys))
+    if (length(area_interchanges) > 0)
+        for i in 1:num_interfaces
+            from_area_name = region_names[interface_regions_from[i]]
+            to_area_name = region_names[interface_regions_to[i]]
+            a_i = filter(
```

Comment:
This is O(n^2) in the number of interfaces if I'm reading this correctly. Perhaps create a dictionary that maps from_area_name, to_area_name to the corresponding vector of area interchanges. I would expect problems with > 100 areas, which doesn't sound completely unreasonable.